### PR TITLE
Fix length 2d land

### PR DESCRIPTION
**Pull request by LePetitTim**
(_No comment_)

### CODE REVIEW

File: geotrek/land/tests/test_views.py [hunk 1]
```diff
@@ -87,7 +87,8 @@ def get_expected_datatables_attrs(self):
         return {
             'id': self.obj.pk,
             'length': round(self.obj.length, 1),
-            'physical_type': self.obj.physical_type_display
+            'physical_type': self.obj.physical_type_display,
+            'length_2d': round(self.obj.length, 1),
```
Comment (submarcos) [inline geotrek/land/tests/test_views.py:91]:
```suggestion
            'length_2d': self.obj.length_2d_display,
```

File: geotrek/land/tests/test_views.py [hunk 2]
```diff
@@ -113,6 +114,7 @@ def get_expected_datatables_attrs(self):
             'id': self.obj.pk,
             'land_type': self.obj.land_type_display,
             'length': round(self.obj.length, 1),
+            'length_2d': round(self.obj.length, 1),
```
Comment (submarcos) [inline geotrek/land/tests/test_views.py:117]:
```suggestion
            'length_2d': self.obj.length_2d_display,
```

File: geotrek/land/tests/test_views.py [hunk 3]
```diff
@@ -137,7 +139,8 @@ def get_expected_datatables_attrs(self):
         return {
             'id': self.obj.pk,
             'length': round(self.obj.length, 1),
-            'organization': self.obj.organization_display
+            'organization': self.obj.organization_display,
+            'length_2d': round(self.obj.length, 1),
```
Comment (submarcos) [inline geotrek/land/tests/test_views.py:143]:
```suggestion
            'length_2d': self.obj.length_2d_display,
```

File: geotrek/land/tests/test_views.py [hunk 4]
```diff
@@ -162,7 +165,8 @@ def get_expected_datatables_attrs(self):
...
```
Comment (submarcos) [inline geotrek/land/tests/test_views.py:169]:
```suggestion
            'length_2d': self.obj.length_2d_display,
```

File: geotrek/land/tests/test_views.py [hunk 5]
```diff
@@ -188,4 +192,5 @@ def get_expected_datatables_attrs(self):
             'id': self.obj.pk,
             'length': round(self.obj.length, 1),
             'organization': self.obj.organization_display,
+            'length_2d': round(self.obj.length, 1),
```
Comment (submarcos) [inline geotrek/land/tests/test_views.py:195]:
```suggestion
            'length_2d': self.obj.length_2d_display,
```

**Comment by LePetitTim** [comment]
https://github.com/GeotrekCE/Geotrek-admin/issues/3150

**Review by submarcos — APPROVED** [review]

**Pull request merged**